src/app/insights/components/InsightsDisplay.tsx

```diff
@@ -39,8 +116,37 @@ export default function InsightsDisplay() {
 
   return (
     <>
+      <div className="flex text-center w-full m-auto justify-between bg-twd-navbar-background py-2 px-4">
+        {dateOptions.map((dateOption, index) => {
+          const isActive = selectedButton === dateOption;
+          return (
+            <Button
+              key={index}
+              label={dateOption}
+              onClick={() =>
+                handleDateChange(dateOption as keyof typeof dateParams)
+              }
+              className={clsx(
+                "font-normal",
+                isActive && "bg-twd-primary-purple text-white"
+              )}
+            />
+          );
+        })}
+        <Button
+          label="To Date"
+          className={clsx(useNow && "bg-white text-black")}
+          onClick={handleUseNowClick}
+        />
+      </div>
+
       {insights ? (
-        <LineGraph dataArray={insights} />
+        <LineGraph
+          dataArray={insights}
+          startOfRange={startOfRange}
+          endOfRange={endOfRange}
+          selectedButton={selectedButton}
+        />
       ) : (
         <div>Loading insights...</div>
       )}
```

Comment:
I think this is not working, but we are getting the loading message from PlotlyChart.tsx, perhaps instead we can pass a prop for customised loading messages if we want this behaviour? [not within the scope of this PR, will not block merge] we could remove the ternary condition within this PR though [will merge if refused]

Reply:
are you talking about the `loading insights...` message? If so, it is working! Just have to watch the screen carefully as it only pops up for a second

Reply:
Oh I see my bad yes I see it